test/unit/partitioner_gpt_test.py

```diff
@@ -57,3 +54,11 @@ def test_set_flag(self, mock_command):
     def test_set_flag_ignored(self, mock_warn):
         self.partitioner.set_flag(1, 'f.active')
         assert mock_warn.called
+
+    @patch('kiwi.partitioner.gpt.Command.run')
+    def test_set_hybrid_mbr(self, mock_command):
+        self.partitioner.partition_id = 3
+        self.partitioner.set_hybrid_mbr()
+        mock_command.assert_called_once_with(
+            ['sgdisk', '-h', '1:2:3', '/dev/loop0']
```

Comment:
the test mocks the number of partitions to be 3 and expects the sgdisk call in the form above. If you don't see anything wrong in the sgdisk call for the number of 3 partitions we should be fine, shouldn't we ?